app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
Though this PR still configures and uses the silencer - so you intend to show the silenced stack in the UI, but the full stack in the logs?  Should the UI show a silenced stack and a full one - or even just the full one, and remove all silencing?